packages/ako-operator-v2/kbld-config.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kbld.k14s.io/v1alpha1
+kind: Config
+overrides:
+  - image: ako-operator:latest
+    newImage: "projects-stg.registry.vmware.com/tkg/sandbox/ako-operator:v1.7.0_vmware.1"
```

Comment:
What is this line for?

Reply:
We don't build ako-operator image inside tanzu-framework, it's a different repo, so it doesn't have the step to replace the `kbld-config` with real image after docker image is generated. For example, when building tanzu-addon-manager package, it will first build the image, then replace the kbld-config with that newly built image. 
If we don't add this hardcoded image, when running the `make build` target, it will fail because `newImage` is empty.
Downstream is different, bolt will just replace the image with ako-operator component's reference. But to get it work on upstream, I need to hardcode something here